geospaas_rest_api/tests.py

```diff
@@ -1,3 +1,146 @@
 from django.test import TestCase
+from django.test import Client
+from geospaas.vocabularies.models import Platform, Instrument 
+from geospaas.vocabularies.models import Parameter, DataCenter
+import json
 
 # Create your tests here.
+class DatasetURITests2(TestCase):
```

Comment:
Please don't add unnecessary empty lines!

Remove empty lines between class definition and the first method.
Replace three empty lines between class definitions with two.
Replace empty line after method definition with a docstring. Docstrings should briefly tell what does the test do. E.g.:
```
def test_api_initial_calls(self):
    """ Shall send request to API root and receive 200 response """
```
See this:
https://github.com/nansencenter/django-geo-spaas/blob/290758f2afde01f615f876c8e950d50dd99cf120/geospaas/vocabularies/tests.py

as a good example of empty lines and docstrings.

Reply:
... and don't add an empty line after docstring.